src/components/Inngest.ts

```diff
@@ -84,6 +85,8 @@ export class Inngest<
 
   private readonly fetch: FetchT;
 
+  public readonly logger: Logger;
```

Comment:
Would be great to make this `private`; the fewer options the user sees when typing `inngest.`, the better.
```suggestion
  private readonly logger: Logger;
```
This may change some other code to require `this.#client["logger"]` instead of `this.#client.logger`.